docs/storage_v2/built_in_features/supercontract/lifecycle.md

```diff
@@ -0,0 +1,53 @@
+---
+id: lifecycle
+title: Supercontract Lifecycle
+sidebar_label: Lifecycle
+---
+
+Any [SC](overview.md) lifecycle contains the next steps:
+
+1. Preparation;
+2. The main steps of execution;
+3. Obtaining results and rewarding.
+4. End of [SC](overview.md)
+
+## Preparation
+
+To work with any [SC](overview.md), you need a [Drive](../drive/overview.md). When it will be done the [owner](../../roles/owner.md) must have some [SC](overview.md) file with [**_\*.wat_**](sc_files.md) extension that will be uploaded to the [Drive](../drive/overview.md). When the file is uploaded on a [Drive](../drive/overview.md) and downloaded by [Storage Replicator Nodes](../../roles/replicator.md) the [client](../../roles/owner.md) can deploy it.
+
+## The main steps of execution
+
+### Deploy
```

Comment:
Need to check. but as I remember in the current realisation when a contractDeployTransaction sent the drive stops be Drive and became SC. That means an owner cannot make changes on drive.